### PR TITLE
Enable prometheus metrics for druid services.

### LINKED ISSUE #10: Monitoring
Metrics can either be written to a log file or posted to an HTTP endpoint. To get that to work with Prometheus, we should use [Druid Exporter](https://github.com/opstree/druid-exporter), deploy it with druid and use that for metrics.

Once we run on docker that should be easy to package together so I'm blocking this issue on the docker issue.

### PR DESCRIPTION
**Pull request by razvan**
## Description

Fixes #10 

## Review Checklist
- [x] Code contains useful comments
- [x] (Integration-)Test cases added (or not applicable)
- [x] Documentation added (or not applicable)
- [x] Changelog updated (or not applicable)
- [x] Cargo.toml only contains references to git tags (not specific commits or branches)

### CODE REVIEW

File: rust/operator/src/lib.rs [hunk 1]
```diff
@@ -475,9 +471,14 @@ impl DruidState {
...
-        if let Some(plaintext_port) = plaintext_port {
-            cb.add_container_port(PLAINTEXT, plaintext_port.parse()?);
+        if let Some(plaintext_port) = validated_config
+            .get(&PropertyNameKind::File(RUNTIME_PROPS.to_string()))
+            .and_then(|props| props.get(DRUID_PLAINTEXTPORT))
+            .map(|port| port.parse::<i32>().unwrap())
+        {
+            cb.add_container_port(CONTAINER_PLAINTEXT_PORT, plaintext_port);
         }
+
```
Comment (maltesander) [inline rust/operator/src/lib.rs:481]: For our current monitoring approach the metrics port should be added here as well?

**Review by maltesander — CHANGES_REQUESTED** [review]
Just two questions?

File: deploy/config-spec/properties.yaml [hunk 2]
```diff
@@ -12,6 +12,33 @@ spec:
...
+        - name: "coordinator"
+          required: true
+        - name: "historical"
+          required: true
+        - name: "middleManager"
+          required: true
+        - name: "router"
+          required: true
+      asOfVersion: "0.22.0"
+      description: "The port where Prometheus metrics are exposed."
```
[the remark this answers is not in the document]
Comment (razvan) [reply deploy/config-spec/properties.yaml:40]: For the stackable platform I think we always want metrics enabled.

File: rust/operator/src/lib.rs (hunk 1 above)
Comment (razvan) [reply rust/operator/src/lib.rs:482]: Done.

**Review by maltesander — APPROVED** [review]
LGTM.

**Pull request merged**